website.i2pd.i2pd.appdata.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Viktor Villainov <supervillain@riseup.net> -->
+<component type="desktop">
+  <id>website.i2pd.i2pd</id>
```

Comment:
Below this add `<launchable type="desktop-id">website.i2pd.i2pd.desktop</launchable>`